apis/infix/atrium-api-infix/src/commonMain/kotlin/ch/tutteli/atrium/api/infix/en_GB/mapLikeToCntainInAnyOrderOnlyCreators.kt

```diff
@@ -81,11 +81,17 @@ inline infix fun <K, reified V : Any, T : MapLike> EntryPointStep<K, out V?, T,
  * @return an [Expect] for the subject of `this` expectation.
  *
  * @since 0.15.0
+ *
+ * @sample ch.tutteli.atrium.api.fluent.en_GB.samples.MapLikeToContainInAnyOrderOnlyCreatorSamples.entriesKeyValue
  */
 inline infix fun <K, reified V : Any, T : MapLike> EntryPointStep<K, out V?, T, InAnyOrderOnlySearchBehaviour>.the(
     entries: KeyValues<K, V>
 ): Expect<T> = entries(V::class, entries.toList())
 
+/**
+ *
+ * @sample ch.tutteli.atrium.api.fluent.en_GB.samples.MapLikeToContainInAnyOrderOnlyCreatorSamples.entries
+ */
```

Comment:
wrong file. you are in the api-infix instead of the api-fluent -- overlooked this in the last PR. You need to fix also all other links

Reply:
Great catch! Moved them over to the right module